manifests/m/Mirantis/Lens/Alpha/5.3.2-latest.20211201.1/Mirantis.Lens.Alpha.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Mirantis.Lens.Alpha
```

Comment:
```suggestion
PackageIdentifier: Mirantis.Lens
```

Reply:
It was listed on their Alpha API. Probably, a mistake(?)